tests/manage/pv_services/test_no_space_left.py

```diff
@@ -0,0 +1,144 @@
+"""
+Test to verify PVC behavior when full of data with I/O
+"""
+import logging
+import pytest
+import ocs_ci.ocs.exceptions as ex
+
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_used_space_on_mount_point
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL],
+            marks=pytest.mark.polarion_id("OCS-852")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM],
+            marks=[
+                pytest.mark.polarion_id("OCS-853"),
+                pytest.mark.bugzilla("1745344")
+            ]
+        ),
+    ]
+)
+@tier1
+class TestPVCFullWithIORWO(ManageTest):
+    """
+    Test to verify PVC behavior when full with data
+    """
+    pvc_size_gb = 5
```

Comment:
The main reason for this test is too check if everything keeps functioning when the pvc is full.
1 is too low as it will be filled immediately. 
5 is enough to assess the the reason for the test

Reply:
right, and the IO size is also the same size as the pvc size